.github/workflows/tests.yml

```diff
@@ -2,7 +2,7 @@ name: Tests
 
 on:
   pull_request:
-    branches: [ jlm-with-llvm-16 ]
+    branches: [ jlm-with-llvm-16, jlm-with-llvm-17 ]
```

Comment:
If the job runs on the `jlm-with-llvm-16` branch, should llvm-version: "16" be passed to the Install packages action? Or Is it intended to always use 17?

Reply:
I wanted to be conservative and not kick the `jlm-with-llvm-16` out immediately, but did not think this through properly. Great catch! Fixed.